bin/linter.dart

```diff
@@ -59,12 +59,14 @@ void runLinter(List<String> args, LinterOptions initialLintOptions) {
         negatable: false)
     ..addOption('config', abbr: 'c', help: 'Use configuration from this file.')
     ..addOption('dart-sdk', help: 'Custom path to a Dart SDK.')
-    ..addOption('lints', help: 'A list of lints to run.', allowMultiple: true)
-    ..addOption('packages',
+    ..addOption('lints',
         help:
-            'Path to the package resolution configuration file, which supplies '
-            'a mapping of package names to paths.  This option cannot be used '
-            'with --package-root.')
+            'A list of lints to run. For example: avoid_as,annotate_overrides',
+        allowMultiple: true)
+    ..addOption('packages',
+        help: 'Path to the package resolution configuration file, which\n'
```

Comment:
I wish the args package made line continuations pretty by itself, but it doesn't. This would require understanding the terminal width, which is nontrivial when considering the many terminals. But today the help text looks like:

```
$ dart bin/linter.dart --help
Lints Dart source files and pubspecs.
Usage: linter <file>
-h, --help                             Shows usage information.
-s, --stats                            Show lint statistics.
    --[no-]visit-transitive-closure    Visit the transitive closure of imported/exported libraries.
-q, --[no-]quiet                       Don't show individual lint errors.
    --machine                          Print results in a format suitable for parsing.
-c, --config                           Use configuration from this file.
    --dart-sdk                         Custom path to a Dart SDK.
    --lints                            A list of lints to run.
    --packages                         Path to the package resolution configuration file, which supplies a mapping of package names to paths.  This option cannot be used with --package-root.
-p, --package-root                     Custom package root. (Discouraged.)

For more information, see https://github.com/dart-lang/linter
```